### ISSUE TITLE
issue with createNativeStackNavigator on iOS

### ISSUE THREAD

**Issue by ciocan** [torgeadelin/react-native-animated-nav-tab-bar#71]
When using **createNativeStackNavigator** from **react-native-screens/native-stack**
it seems the animated tab bar overlay is on top of everything and any buttons placed on the screens are not interactive.

I noticed when **enableScreens()** is called the issue starts appearing.

On Android is working fine.

**Comment by torgeadelin** [comment]
Hi, could you post some steps to reproduce, or some source code.

Thanks!

**Comment by ciocan** [comment]
Sure do, I updated your example here:
https://github.com/ciocan/react-native-animated-nav-tab-bar/blob/main/navigation/AppNavigation.tsx

I added: 
```
import { enableScreens } from 'react-native-screens'
...
enableScreens()
...
```

**Comment by torgeadelin** [comment]
Fixed, please update to 3.1.2 and reset cache.
Thanks :)
